telegram/_telegramobject.py

```diff
@@ -211,3 +209,14 @@ def __hash__(self) -> int:
         if self._id_attrs:
             return hash((self.__class__, self._id_attrs))
         return super().__hash__()
+
+    def __getstate__(self):
+        return self.to_dict()
```

Comment:
please check how this treats types apart from int/str/float - e.g. IIRC `to_dict` converts datetimes to float timestamps and init doesn't re-convert 🤔

Reply:
would calling `de_json(states)` instead suffice?

Reply:
should work, I guess so … however again the recursive to_dict/de_json also takes some runtime, that's not really needed here. Maybe we can extract the logic of getting the relevant attributes from `to_dict()` and use it in both places?
in any case, test cases will have to cover such things :)